com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/core/engine/importer/impl/LoadFileImporter.java

```diff
@@ -176,14 +176,15 @@ protected void handleDataFile(
             MigrationDirAndFilesManager mdfm = mrManager.getDirAndFilesMgr();
             String schemaName =
                     config.getSrcCatalog().getDatabaseType().isSupportMultiSchema()
+                                    && !(stc.getOwner() == null)
```

Comment:
`!=` 을 사용하는 것이 좋아보입니다.

```suggestion
                                    && stc.getOwner() != null
```

Reply:
확인했습니다 내용을 수정합니다